data/strings/strings.txt

```diff
@@ -26952,7 +27063,7 @@ fa = دوباره نپرس
     el = Σύμφωνα με %d κριτικές
     sk = Na základe %d recenzií
     sw = Kulingana na maoni %d
-
+    fa = براساس %d ارزیابی
```

Comment:
Please leave the empty line before the next block.

Reply:
fixed